CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 1.0.1 (Next)
 
 * Your contribution here.
+* [#65](https://github.com/mongoid/mongoid-locker/pull/65): Drop `mongoid-compatibility` gem dependency - [@dks17](https://github.com/dks17).
```

Comment:
I did merge, but now I realize this is incorrect. The gem did not depend on mongoid-compatibility. This was only used in tests.

Reply:
Oh nvm I am wrong.